tests/utils.py

```diff
@@ -18,6 +18,13 @@
 packageName = 'ga4gh'
 
 
+def zipSameLength(*iterables):
+    length = len(iterables[0])
+    for iterable in iterables[1:]:
+        assert len(iterable) == length
```

Comment:
Technically, this is assuming that the iterable is a list/tuple/etc as you can't can't call `len` on a generator. See 
https://docs.python.org/3/library/itertools.html#itertools.zip_longest for clues on how to do it generally.